function catchAll(Context cx) returns error? {
    scopeFn(cx);
}
